decidim-admin/app/commands/decidim/admin/update_organization_appearance.rb

```diff
@@ -68,8 +68,7 @@ def appearance_attributes
           remove_official_img_header: form.remove_official_img_header,
           official_img_footer: form.official_img_footer,
           remove_official_img_footer: form.remove_official_img_footer,
-          official_url: form.official_url,
-          show_statistics: form.show_statistics
+          official_url: form.official_url
```

Comment:
Codecov complains about an added line without tests but it's actually a removed line :/